### PR TITLE
INT - B-21905 - Rename NTS shipment type

### PR DESCRIPTION
**Pull request by taeJungCaci**
## Summary

This is to change current NTS shipment type, `HHG_INTO_NTS_DOMESTIC` to `HHG_INTO_NTS` since we will be using these for international shipments. This change applies to everywhere in the app including the enum in the DB.

### How to test

1. Make sure your current NTS shipment still works after running migration on this branch if you had any.
2. Make sure you can see the enum changed to `HHG_INTO_NTS` in the DB for `shipment_type` in `mto_shipments` table.
3. Make sure you can create and go through the normal flow with new or existing NTS shipment.
4. Go to prime sim -> find NTS shipment -> and verify that you see `HHG_INTO_NTS` as the shipment type in prime GET api.
5. Make sure nothing DB related is breaking  🤣 

## Screenshots
image
image

### CODE REVIEW

File: migrations/app/schema/20241216170325_update_nts_enum_name.up.sql
[the remark this answers is not in the document]
Comment (taeJungCaci) [reply migrations/app/schema/20241216170325_update_nts_enum_name.up.sql:1]: Oh yes! Thanks for catching that. I had that in the file and then I edited and forgot to put it back in 😆

[the remark this answers is not in the document]
Comment (taeJungCaci) [reply migrations/app/schema/20241216170325_update_nts_enum_name.up.sql:1]: Okay - changes are in. Should be good now.

[the remark this answers is not in the document]
Comment (brianmanley-caci) [reply migrations/app/schema/20241216170325_update_nts_enum_name.up.sql:1]: Looks good now 👍 

Screenshot 2024-12-17 at 3 48 17 PM

**Comment by brianmanley-caci** [comment]
The Happo changes appear to be from existing change in integration (mostly city finder), so I approved them.

**Review by brianmanley-caci — APPROVED** [review]
1. Existing NTS shipments displaying correctly.
2. mto_shipment_type enum has 'HHG_INTO_NTS' value.
3. Created, submitted, and approved a new NTS shipment without any errors.
4. Called getMoveTaskOrder on an NTS shipment, and I can see the 'HHG_INTO_NTS' shipment type in the response.
5. No DB issues encountered.
🎉 🎉 🎉

**Comment by taeJungCaci** [comment]
> 1. Existing NTS shipments displaying correctly.
> 2. mto_shipment_type enum has 'HHG_INTO_NTS' value.
> 3. Created, submitted, and approved a new NTS shipment without any errors.
> 4. Called getMoveTaskOrder on an NTS shipment, and I can see the 'HHG_INTO_NTS' shipment type in the response.
> 5. No DB issues encountered.
>    🎉 🎉 🎉

Thank you Mr. AllInOne!! lol 🤣

**Review by KonstanceH — DISMISSED** [review]
Tested creating and updating NTS shipment while checking payload from customer to officer and did static review. Everything is working/looking good to me.
LGTM!

**Comment by taeJungCaci** [comment]
Merged in latest IntegrationBranch and had to resolve conflict for migration_manifest. And I need approvals again 🥲 @KonstanceH @brianmanley-caci

**Review by brianmanley-caci — APPROVED** [review]

**Review by KonstanceH — APPROVED** [review]

**Pull request merged**